.github/PULL_REQUEST_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,45 @@
+## Description
+<!--
+Thank you so much for your PR!  To help us review your contribution, please
+consider the following points:
+
+- The PR title should summarize the changes, for example "Add new group argument for the
+  pair plot".  Avoid non-descriptive titles such as "Addresses issue #348". If your pull 
+  request addresses an issue, please use the pull request title to describe 
+  the issue and mention the issue number in the pull request description.
+
+- The description should provide at least 1-2 sentences describing the pull request
+  in detail (Why is this change required?  What problem does it solve?) and
+  link to any relevant issues. If modifying a plot, render your plot to inspect for changes 
+  and copy image in the pull request message on Github
+
+- Please prefix the title of incomplete contributions with [WIP] (to indicate a work in 
+  progress). WIPs may be useful to (1) indicate you are working on something to avoid 
+  duplicated work, (2) request broad review of functionality or API, or (3) seek collaborators.
+-->
+
+## Checklist
+<!-- Feel free to remove check-list items aren't relevant to your change -->
+
+- [ ] Does the PR follow [official](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#pull-request-checklist) 
+      PR format?
+- [ ] Is the documentation [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant?
+- [ ] Is fix listed in the [Maintenance and fixes](https://github.com/arviz-devs/arviz/blob/master/CHANGELOG.md#maintenance-and-fixes) 
+      section of the changelog?
```

Comment:
Here this should refer to the Documentation subsection: https://github.com/arviz-devs/arviz/blob/master/CHANGELOG.md#documentation instead of the maintenance and fixes.

Reply:
@OriolAbril Yes, I understand. Sorry, I have not seen the structure of `changelog.md` before.